src/Pages/PopularRepositoriesTable/PopularRepositoriesTable.tsx

```diff
@@ -355,12 +372,11 @@ const PopularRepositoriesTable = () => {
             <InputGroupItem isFill>
               <TextInput
                 isDisabled={isLoading}
-                type='text'
                 id='search'
-                ouiaId='popular_filter_search'
+                ouiaId='filter_search'
                 placeholder='Filter by name/url'
-                value={searchValue}
-                onChange={(_event, val) => setSearchValue(val)}
+                value={searchQuery}
```

Comment:
you've changed this value to a new one you've created, but the API is still using the old one. 

This makes it so that when you type into the search box, the api is not filtered. 

So at the moment, the searchbox does not function as intended.